src/pages/steps/AdditionalInformationStep/AdditionalInformationStep.tsx

```diff
@@ -216,20 +225,26 @@ const AdditionalInformationStep = ({
                 title={
                   <TabTitle
                     field={field}
-                    isCompleted={completedFields[field]}
+                    validationStatus={validatedFields[field]}
                   />
                 }
               >
                 <TabContent
                   minHeight="450px"
                   offerId={offerId}
                   scope={scope}
-                  onChangeCompleted={(isCompleted) => {
-                    if (completedFields[field] === isCompleted) return;
+                  onValidationChange={(status) => {
+                    if (typeof status === 'boolean') {
+                      status = status
+                        ? ValidationStatus.SUCCESS
+                        : ValidationStatus.NONE;
+                    }
```

Comment:
I would only accept the `ValidationStatus` for `onValidationChange` then you can pass `ValidationStatus.SUCCESS` instead of `true`and `ValidationStatus.NONE` instead of `false`

Reply:
That's what I started to do but it was a heavier change so since this is my first semi-big PR I was wary of changing everything :D